source/_components/yeelight.markdown

```diff
@@ -73,6 +73,10 @@ devices:
           required: false
           type: boolean
           default: false
+        nightlight_switch_type:
+          description: Adds another entity, to control nightlight mode (for models that supports it). Currently, only `light` is supported. It will create 2 light entities, one for normal light mode and second for nightlight mode. They are mutually exclusive.
```

Comment:
Considering these are mutually exclusive, shouldn't this be a boolean?

Reply:
My idea was to provide way for more nightlight switch type, like switch ( keep 1 light entity but add switch for normal / nighlight mode and there was idea, on forum, to use brightness slider to switch between that mode. So I've thought it would be good to leave room for possible future improvements / other ideas.

Reply:
Ok I think that's fine. As I just merged the parent, let's merge this too.